df/plugin.go

```diff
@@ -412,14 +422,18 @@ func (dfs *dfStats) collect(procPath string, excluded_fs_names []string, exclude
 		dfm.Filesystem = rightFields[1]
 		dfm.FsType = rightFields[0]
 		dfm.UnchangedMountPoint = leftFields[4]
-		if leftFields[4] == "/" {
-			dfm.MountPoint = "rootfs"
+		if keep_original_mountpoint {
```

Comment:
@jcooklin This was definitely done on purpose to be conservative with default behaviour. However I would be very happy to change this and make the default behaviour now be to keep the original mount point
New PR version will follow accordingly